### ISSUE TITLE
Add friction to switching between zoom-level-limited geolevels

### ISSUE THREAD

**Issue by tgilcrest** [PublicMapping/districtbuilder#268]
As explained in #267, some geolevels are limited to certain zoom levels. Because these zoom-level-limited geolevels are generally more advanced, we should add some friction/explanation so less experienced users don't get confused.

AC:
1. Users can only switch to zoom-level-limited geolevels if they 1) are zoomed in far enough and 2) if they have no active selection in other geolevels
2. If either of those criteria are not met, the button to switch to the geolevel is deactivated. When they are met the button is activated.
3. There is a tooltip when hovering over the geolevel to indicate which of these criteria are met/not, similar to the password validation tooltip
4. If the user selects the ZLL geolevel and makes a selection, the non-ZLL geolevels buttons are deactivated.
5. There is a tooltip over the non-ZLL geolevels that indicates the criteria for switching to them (no active selections), similar to 3.

@jfrankl has a design for how this should look.

**Comment by jfrankl** [comment]
Here are some related mockups: https://share.goabstract.com/0446558a-4a08-43dc-8103-695ff2360f46

**Comment by tgilcrest** [comment]
@pcaisse are you including the [modal shown here](https://share.goabstract.com/0446558a-4a08-43dc-8103-695ff2360f46?collectionLayerId=3d8cce74-5f15-41c6-a764-3e60e788f2cc&mode=design), which shows up the first time the user selects blocks or should that be a separate task?

**Comment by pcaisse** [comment]
> @pcaisse are you including the [modal shown here](https://share.goabstract.com/0446558a-4a08-43dc-8103-695ff2360f46?collectionLayerId=3d8cce74-5f15-41c6-a764-3e60e788f2cc&mode=design), which shows up the first time the user selects blocks or should that be a separate task?

I wasn't planning on it since I didn't see it in the acceptance criteria. I think it might make sense to do separately since it isn't strictly required for this and there might be some complexity around when it's shown (only once ever per user vs. once ever on device vs once per session).

**Comment by pcaisse** [comment]
Updated mocks for tooltip
![image (4)](https://user-images.githubusercontent.com/2926237/89816443-94c52b80-db14-11ea-911d-f9a8b6bd09ec.png)
